docs/content/users/usage/commands.md

```diff
@@ -180,6 +180,7 @@ Flags:
 * `--dbimage-extra-packages`: A comma-delimited list of Debian packages that should be added to db container when the project is started.
 * `--default-container-timeout`: Default time in seconds that DDEV waits for all containers to become ready on start. (default `120`)
 * `--disable-settings-management`: Prevent DDEV from creating or updating CMS settings files.
+* `--disable-upload-dirs-warning`: Don't issue a warning when a project does not have `upload_dirs` set but is using `performance_mode: mutagen`.
```

Comment:
Use a more active verb, and flip the negative and affirmative so it’s easier to follow:

```suggestion
* `--disable-upload-dirs-warning`: Suppresses warning when a project is using `performance_mode: mutagen` but does not have `upload_dirs` set.
```